src/database/mongodb.ts

```diff
@@ -0,0 +1,14 @@
+import * as mongoose from 'mongoose';
+
+const ENDPOINT = process.env.ENDPOINT_DB || 'localhost:27018';
```

Comment:
Should be 27017 if you refer to docker-compose.yml

Reply:
mb